pycbc/workflow/grb_utils.py

```diff
@@ -560,30 +560,37 @@ def make_pygrb_plot(workflow, exec_name, out_dir,
     return node, node.output_files
 
 
-def make_info_table(workflow, out_dir, tags=None):
-    """Setup a job to create an html snippet with the GRB trigger information.
+def make_info_table(workflow, exec_name, out_dir, in_files=None, tags=None):
+    """
+    Setup a job to create an html snippet with the GRB trigger information
+    or exlusion distances information.
     """
 
+    # Organize tags
     tags = [] if tags is None else tags
-
-    # Executable
-    exec_name = 'pygrb_grb_info_table'
+    grb_name = workflow.cp.get('workflow', 'trigger-name')
+    extra_tags = ['GRB'+grb_name]
 
     # Initialize job node
-    grb_name = workflow.cp.get('workflow', 'trigger-name')
-    extra_tags = ['GRB'+grb_name, 'INFO_TABLE']
     node = PlotExecutable(workflow.cp, exec_name,
                           ifos=workflow.ifos, out_dir=out_dir,
                           tags=tags+extra_tags).create_node()
 
     # Options
-    node.add_opt('--trigger-time', workflow.cp.get('workflow', 'trigger-time'))
-    node.add_opt('--ra', workflow.cp.get('workflow', 'ra'))
-    node.add_opt('--dec', workflow.cp.get('workflow', 'dec'))
-    node.add_opt('--sky-error', workflow.cp.get('workflow', 'sky-error'))
-    node.add_opt('--ifos', ' '.join(workflow.ifos))
+    if exec_name == 'pygrb_grb_info_table':
+        node.add_opt('--trigger-time',
+                     workflow.cp.get('workflow', 'trigger-time'))
+        node.add_opt('--ra', workflow.cp.get('workflow', 'ra'))
+        node.add_opt('--dec', workflow.cp.get('workflow', 'dec'))
+        node.add_opt('--sky-error', workflow.cp.get('workflow', 'sky-error'))
```

Comment:
Why do these need to be read from `cp.get`? The all-sky workflow has numerous cases where options are shared between codes, and the config files have syntax to support this .... The idea being to avoid these setup codes being unecessarily complex and having the configs specifying, as much as possible, what arguments codes will run with.